tests/data/sample_pointing_videos_161215_15_34_21.sig2

```diff
@@ -1,441 +1,443 @@
 Generating signature for sample_pointing_videos_161215_15_34_21.nwb
 
+/
+	@.specloc: type=Reference val="->/specifications"
 /acquisition/EyeCam
-	@comments: type=str val="Frame rate 1/30 s"
+	@comments: type=str val="Frame rate 30 fps"
 	@description: type=str val="Video recording of mouse behaviour."
-/acquisition/EyeCam/bits_per_pixel: dtype=int32 shape=() val="8"
-/acquisition/EyeCam/dimension: dtype=int32 shape=(2,) val="0x4f70110"
+/acquisition/EyeCam/dimension: dtype=int64 shape=(2,) val="0x9ef0110"
```

Comment:
There are quite a few things that were 32bit and are now 64bit which might contribute to an increase in file size. Although mostly they seem to be small datasets so shouldn't be significant.

Reply:
True. I'm not sure how much control we have over the number of bits used when writing, but maybe we should experiment with casting some things.